YDS-SwiftUI/Source/Component/YDSListItem.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  YDSListItem.swift
+//  YDS-SwiftUI
+//
+//  Created by 김수민 on 2023/11/04.
+//
+
+import SwiftUI
+
+struct YDSListItem: View {
+    var icon: Bool?
+    var toggle: Bool?
+    var text: String?
+    
+    @Binding var isOn: Bool
+    
+    public init(text: String?, icon: Bool?, toggle: Bool?, isOn: Binding<Bool>) {
+        self.text = text
+        self.icon = icon
+        self.toggle = toggle
+        self._isOn = isOn
+    }
+    public var body: some View {
+        HStack {
+            if let text = text{
+                Text(text)
+                    .foregroundColor(YDSColor.textSecondary)
+                    .padding(.horizontal, 20)
+                    .padding(.vertical, 20)
+                    .frame(height: 23)
+            }
+            Spacer()
+            if let icon = icon, icon {
+                YDSIcon.arrowRightLine
+                    .padding(.horizontal, 20)
+                    .padding(.vertical, 20)
```

Comment:
가로 세로 같은 패딩값을 주니까
```suggestion
                    .padding(20)
```
이렇게 하나로 바꾸는건 어때요?

Reply:
수정했습니다!